google-cloud-spanner/lib/google/cloud/spanner/database.rb

```diff
@@ -259,11 +262,34 @@ def ddl force: nil
         #
         #   database.update statements: [add_users_table_sql]
         #
-        def update statements: [], operation_id: nil
+        # @example
+        #   require "google/cloud/spanner"
+        #
+        #   spanner = Google::Cloud::Spanner.new
+        #   database = spanner.database "my-instance", "my-database"
+        #
+        #   create_proto_bundle_sql = %q(
```

Comment:
I know the other examples use `%q` but I think it's better to use heredocs (e.g. `<<~SQL`...) for multi-line literal text, like you do in the tests, because it does better with newlines and indentation.